### PR TITLE
Fixes #20864: use report_id for command_execution generic method

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/20864

### CODE REVIEW

**Review by amousset — APPROVED** [review]

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

File: tree/30_generic_methods/command_execution_once.cf [hunk 1]
```diff
@@ -118,11 +118,12 @@ bundle agent command_execution_once(command, ok_codes, until, unique_id)
 
     # Problem when writing the flag
     pass3.reserved.!set_ok::
-      "failure" usebundle => _classes_failure("${old_class_prefix}");
-      "failure" usebundle => _classes_failure("${class_prefix}");
+      "failure"                  usebundle => _classes_failure("${class_prefix}");
+      "${report_data.method_id}" usebundle => _classes_failure("${report_data.method_id}");
```
Comment (amousset) [inline tree/30_generic_methods/command_execution_once.cf:122]: what I did for the methods I migrated was to systematically use the method_id for all usebundle promisers

Comment (ncharles) [reply tree/30_generic_methods/command_execution_once.cf:122]: that's not enough - the method don't report

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

**Pull request merged**